io.github.arunsivaramanneo.GPUViewer.yml

```diff
@@ -3,7 +3,7 @@ runtime: org.freedesktop.Platform
 sdk: org.freedesktop.Sdk
 runtime-version: "18.08"
 finish-args:
-  - --device=dri
+  - --device=all # OpenCL requires /dev/nvidia-uvm
```

Comment:
This sounds like a bug to report to flatpak/flatpak.

Reply:
OpenCL on Nvidia in FDO runtime 18.08 seems broken, but can be fixed locally.

Reply:
> In the runtime files, /etc/OpenCL should point to ../lib/x86_64-linux-gnu/GL/OpenCL, and not to ../lib/GL/OpenCL
> In runtime's metadata, OpenCL/vendors should be added to merge-dirs

Please report that here: https://gitlab.com/freedesktop-sdk/freedesktop-sdk

> The .GL.nvidia-* extension lacks OpenCL symlink to extra/OpenCL

Please report that here: https://github.com/flatpak/freedesktop-sdk-images